components/ILIAS/Mail/classes/Setup/Migration/class.ilUpdateMailTemplatesForMustache.php

```diff
@@ -29,7 +29,7 @@ class ilUpdateMailTemplatesForMustache implements Migration
 
     public function getLabel(): string
     {
-        return 'ilUpdateMailTemplatesForMustache';
+        return "ilUpdateMailTemplatesForMustache";
```

Comment:
Do we really need this change? Is it related to the feature?

And: IMO we can remove the complete migration for ILIAS 10.x (I could change this separately, if you like).